const main = async () => {
  // Test nothing
}

main().catch(e => console.log)